packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.html

```diff
@@ -26,6 +26,41 @@
                                       [color]="chartAssist.palette.standardColors.get(legendSeries.id)">
                     </nui-chart-marker>
                 </nui-rich-legend-tile>
+
+                <div description class="d-flex align-items-center px-2">
+                    <div class="description-container d-flex flex-column justify-content-center">
+                        <div *ngIf="(legendSeries?.link? false : true) else elseLinkHref">
```

Comment:
Could be simplified in this way:

```suggestion
                        <div *ngIf="!legendSeries?.link else elseLinkHref">
```